test/test_helpers/storage.rb

```diff
@@ -25,6 +28,12 @@ module Mock
       DEFAULT_NODES = ["127.0.0.1:7379", "127.0.0.1:7380"].freeze
       private_constant :DEFAULT_NODES
 
+      STORAGE_CLASSES = [
+          ThreeScale::Backend::StorageAsync::Client,
+          ThreeScale::Backend::StorageSync
+      ].freeze
```

Comment:
I'd like to be able to run the tests with sync without loading the async client and its dependencies. Not a big issue, but would be nice to have. Just referring to it here probably requires the interpreter to load up the dependencies.